src/constants/routes.ts

```diff
@@ -0,0 +1,18 @@
+import Page404 from "../pages/404";
+import MachineActions from "../pages/MachineActionsPage";
+import MachineEvents from "../pages/MachineEventsPage";
+
+export const Routes = [
+  {
+    path: "/",

```

Comment:
and paths as separate constants, because you will use this paths not only in this file